src/atoms/google-analytics/google-analytics.tsx

```diff
@@ -24,5 +25,59 @@ const GoogleAnalyticsBase = ({ config }) => {
   return <Analytics type="googleanalytics" targets={gaConfig} />;
 };
 
+const GoogleAnalytics4 = ({config}) => {
+  const Ga4ConfigObject = get(config, ["opts", "featureConfig", "ga4Config"], null);
+  return <Analytics type="googleanalytics" targets={Ga4ConfigObject} config="./ga4.json" data-credentials="include"/>;
+};
+
+const GoogleAnalyticsBase = ({config}) => {
+  const googleAnalyticsTrackingId = config.ampConfig["google-analytics-tracking-id"];
+  if (!googleAnalyticsTrackingId) return null;
```

Comment:
this will not work for ahead where we're taking tracking ID from pagebuilder and not from AMP config. Please also check whether it is being passed from featureConfig